manuscript/01-das-modell-zum-einsatz-bringen.md

```diff
@@ -1,70 +1,65 @@
-Taktik und Strategie müssen kombiniert werden, um erfolgreich zu sein und
-DDD befasst sich sowohl mit taktischem als auch mit strategischem
+Taktik und Strategie müssen kombiniert werden, um erfolgreich zu sein 
+und DDD befasst sich sowohl mit taktischem als auch mit strategischem
 Design.
 
 ## Bounded Context {#bounded-context}
 
 *dt.: Begrenzter Kontext*
 
 *Bei jedem großen Projekt sind mehrere Modelle im Spiel.* Sie
-entstehen aus vielen Gründen.  Zwei Subsysteme bedienen in der Regel
+entstehen aus vielen Gründen. Zwei Subsysteme bedienen in der Regel
 sehr unterschiedliche Benutzergruppen mit unterschiedlichen Aufgaben,
-wobei unterschiedliche Modelle sinnvoll sein können.  Selbständig
+wobei unterschiedliche Modelle sinnvoll sein können. Selbständig
 arbeitende Teams können das gleiche Problem durch mangelnde
-Kommunikation auf unterschiedliche Weise lösen.  Auch können die
-benutzten Werkzeuge
-unterschiedlich sein, so dass Programmcode nicht gemeinsam
-genutzt werden kann.
+Kommunikation auf unterschiedliche Weise lösen. Auch können die
+benutzten Werkzeuge unterschiedlich sein, so dass Programmcode nicht 
+gemeinsam genutzt werden kann.
 
 Mehrere Modelle sind unvermeidlich, aber wenn Code, der auf
 verschiedenen Modellen basiert, kombiniert wird, wird Software
-fehlerhaft, unzuverlässig und schwer verständlich.  Die Kommunikation
-zwischen den Teammitgliedern wird konfus.  Oft ist unklar, in
+fehlerhaft, unzuverlässig und schweirig verständlich. Die Kommunikation
+zwischen den Teammitgliedern wird konfus. Oft ist unklar, in
 welchem Zusammenhang ein Modell nicht angewendet werden soll.
 
-Ausdrücke aus dem Modells haben, wie jede andere Formulierung auch,
-nur im [Context](#context) eine Bedeutung.
+Ausdrücke aus dem Modell haben, wie jede andere Formulierung auch,
+nur im [Kontext](#context) eine Bedeutung.
 
 Daher:
 
-**Definiere explizit den Kontext, in dem ein Modell angewendet wird.
-Lege explizit die Grenzen bezüglich  Teamorganisation, 
-Verwendung innerhalb bestimmter Teile der Anwendung und
-physischen Umsetzungen wie Codebasen und Datenbankschemata fest.  Wende
+**Definiere explizit den Kontext, in dem ein Modell Anwendung findet.
+Lege explizit die Grenzen bezüglich Teamorganisation, Verwendung 
+innerhalb bestimmter Teile der Anwendung und
+physischen Umsetzungen wie Codebasen und Datenbankschemata fest. Wende
 [Continuous Integration](#continuous-integration) an, um
-Modellkonzepte und -begriffe innerhalb
-dieser Grenzen strickt konsistent zu halten, lass dich aber nicht von
-Problemen außerhalb ablenken oder verwirren.  Standardisiere
-einen Entwicklungsprozess innerhalb des Kontextes, der nicht
-anderswo verwendet werden muss.**
+Modellkonzepte und -begriffe innerhalb dieser Grenzen strikt konsistent 
+zu halten, lass dich aber nicht von
+Problemen außerhalb ablenken oder verwirren. Standardisiere
+einen einzigen Entwicklungsprozess innerhalb des Kontextes, der
+anderswo nicht verwendet werden muss.**
```

Comment:
"which need not be used elsewhere" - meint:

a) der anderswo nicht (zwingend) verwendet werden muss --> kein globaler Entwicklungsansatz
b) der anderswo nicht verwendet werden darf

Meine Interpretation is a)